docs/hardware_setup.md

```diff
@@ -119,6 +123,10 @@ To build JetBot you'll need the following tools
 
 ### Step 7 - Remove Jetson Nano module from developer kit
 
+!!! check ""
```

Comment:
I think we can assume B.O.M will guide users on the choice between wifi module for 2GBvs.4GB.  So here we likely just need to mention USB wiFi.

!!! attention ""

    Skip this step if you are using a USB WiFi dongle instead of the M.2 WiFi module